cmd/auth-clientCredentials.go

```diff
@@ -48,24 +50,20 @@ func auth_clientCredentials(cmd *cobra.Command, args []string) {
 
 	// check if we have a clientSecret in the keyring, if a null value is passed in
 	if clientSecret == "" {
-		clientSecret, err = handlers.GetClientSecretFromCache(clientID)
+		clientSecret, err = handlers.GetClientSecretFromCache(host, clientID)
 		if err == nil || clientSecret == "" {
 			cli.ExitWithError("Please provide required flag: (client-secret)", errors.New("no client-secret found"))
 		} else {
 			slog.Debug("Retrieved stored client-secret from keyring")
 		}
 	}
 
-	tok, err := handlers.GetTokenWithClientCreds(cmd.Context(), clientID, clientSecret, handlers.TOKEN_URL, noCacheCreds, tlsNoVerify)
-	if err != nil {
+	slog.Debug("Attempting to login with client credentials", slog.String("client-id", clientID))
+	if err := handlers.GetTokenWithClientCreds(cmd.Context(), host, clientID, clientSecret, tlsNoVerify); err != nil {
 		cli.ExitWithError("An error occurred during login. Please check your credentials and try again", err)
 	}
 
-	if !noCacheCreds {
-		fmt.Println(cli.SuccessMessage("Successfully logged in with client ID and secret"))
-	} else {
-		fmt.Print(tok.AccessToken)
```

Comment:
Should we preserve the ability to print the access token so it could be read into authenticated actions when the SDK supports simply passing an OIDC AT? Maybe something like the below:

```shell
echo secret.txt | otdfctl encrypt --with-access-token $(otdfctl auth client-credentials --no-cache)
```

Reply:
@jakedoublev and I decided to do this as a separate PR when SDK supports access token.